oval_graph/command_line_client/__init__.py

```diff
@@ -0,0 +1,3 @@
+'''
+    This module is intended for processing command parameters.
```

Comment:
You probably don't mean *module*, but a *package*. But again, I wouldn't document this, one can find out what is where quickly with low effort, and without risk of using outdated information.